opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -349,6 +362,326 @@ def collect(
         return current_point
 
 
+# pylint: disable=protected-access
+class _ExponentialBucketHistogramAggregation(_Aggregation[HistogramPoint]):
+    # _min_max_size and _max_max_size are the smallest and largest values
+    # the max_size parameter may have, respectively.
+
+    # _min_max_size is is the smallest reasonable value which is small enough
+    # to contain the entire normal floating point range at the minimum scale.
+    _min_max_size = 2
```

Comment:
Is this limit part of the specification?

Reply:
It does not need to be, it is a mathematical consequence of representing floating point numbers in IEEE 754 format. I added a [test case](https://github.com/open-telemetry/opentelemetry-python/pull/2964/commits/fccfccbd5c4cdba78ec5a60d6c57bb698653889e) that show how this happens.

Reply:
I see this as a minimum viable size for a histogram to be useful. A size=1 or size=0 histogram is not sensible. With minimum scale of -10, we get one bucket for (0x1p-1024, 1], and one bucket for (1, 0x1p1024]. So I see this as a consequence of the minimum scale being set correctly.